src/azure-cli/azure/cli/command_modules/keyvault/tests/hybrid_2018_03_01/test_keyvault_commands.py

```diff
@@ -195,9 +194,9 @@ def test_keyvault_key(self, resource_group):
         self.kwargs['key_file'] = key_file
         self.cmd('keyvault key backup --vault-name {kv} -n {key} --file {key_file}')
         self.cmd('keyvault key delete --vault-name {kv} -n {key}')
-        time.sleep(10)
+        time.sleep(200)
```

Comment:
Sleeping for 200s makes live test time consuming. Any reason for the change? How about adding some retry logic, like the one in `role` module:

https://github.com/Azure/azure-cli/blob/11347b73999dc9b9d1c6f924fb42e3d20b34a767/src/azure-cli/azure/cli/command_modules/role/tests/util.py#L12-L26

Reply:
If the deletion has not finished, the later commands will fail.

The retry logic is more elegant, I can refine the code in another PR. For this one, I want it get merged **ASAP** since I have resolved conflicts for 3 times. Tough time😞